deploy/kubernetes/kiali.yaml

```diff
@@ -23,6 +23,24 @@ spec:
     app: kiali
     version: ${VERSION_LABEL}
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kiali
```

Comment:
This is broken - this duplicates the same service name as the main Kiali service. We need to fix this - deployment on k8s is going to break otherwise. call this something like "kiali-jaeger"